test-site/src/pages/UniversalPage.tsx

```diff
@@ -1,12 +1,44 @@
-import { useAnswersActions } from '@yext/answers-headless-react';
+import { provideAnswersHeadless, useAnswersActions } from '@yext/answers-headless-react';
 import {
   DirectAnswer,
   ResultsCount,
   SearchBar,
   SpellCheck,
-  UniversalResults
+  UniversalResults,
+  EntityPreviews,
+  VisualAutocompleteConfig
 } from '@yext/answers-react-components';
 import { useLayoutEffect } from 'react';
+import { config } from '../config';
+
+
+const visualAutocompleteConfig: VisualAutocompleteConfig = {
+  entityPreviewSearcher: provideAnswersHeadless({
+    ...config,
+    headlessId: 'visual-autocomplete'
+  }),
+  restrictVerticals: ['people'],
+  renderEntityPreviews: isLoading => (
+    <div className={isLoading ? 'opacity-50' : ''}>
+      <EntityPreviews verticalKey='people'>
+        {results => (
+          <div className='flex ml-4 mt-1'>
+            {results.map((r, index) =>
+              <div
```

Comment:
out of curiosity, does the dropdown and arrow navigation work properly like this without wrapping in `DropdownItem`?

Reply:
Hmm I didn't intend for the visual autocomplete results to be navigable here, they were just supposed to be a preview of the universal results. Is it more common for people to want them as navigable?

Reply:
Ah ok, not sure which is more common, so as long as they appear in the dropdown properly, that's fine! I just wasn't sure if that was the case because I thought I remembered someone, maybe Aaron, having some issues when they didn't wrap in `DropdownItem`